### ISSUE TITLE
Color contrast too low on Getting Started text

### ISSUE THREAD

**Issue by yiskah** [tootsuite/mastodon#532]
The font color for the `<p>/<span>` under `<div class="static-content getting-started">` is too low to meet WCAG accessibility standards. Especially given that it overlaps with the elephant graphic (which looks bad on every screen I have.)

my suggested fix would be on line 653 in components.scss:

`.getting-started {
  box-sizing: border-box;
  overflow-y: auto;
  padding-bottom: 235px;
  background: image-url('mastodon-getting-started.png') no-repeat 0 100% local;
  height: 100%;
  p {font-color: $color2;}
}` 

The $color2 variable passes AAA WCAG contrast standards when laid over $color1. You could also use $color3 but it only passes AA and like, there's no reason I'm aware of not to go for AAA accessibility on something as simple as text color contrast.

EDIT: created a corresponding PR for this: https://github.com/tootsuite/mastodon/pull/534

**Comment by yiskah** [comment]
My  PR got merged all should be good
